mesonbuild/scripts/depfixer.py

```diff
@@ -391,12 +385,10 @@ def remove_rpath_entry(self, entrynum: int) -> None:
         return None
 
 def fix_elf(fname: str, rpath_dirs_to_remove: T.Set[bytes], new_rpath: T.Optional[bytes], verbose: bool = True) -> None:
-    with Elf(fname, verbose) as e:
-        if new_rpath is None:
-            e.print_rpath()
-            e.print_runpath()
-        else:
-            e.fix_rpath(rpath_dirs_to_remove, new_rpath)
+    if new_rpath is not None:
+        with Elf(fname, verbose) as e:
+            # note: e.get_rpath() and e.get_runpath() may be useful
```

Comment:
So for debug purposes you'd edit the script and use these functions? Maybe it should be printed `if verbose`?

Reply:
Just insert `breakpoint()` in the appropriate place and explore from there… There are 5 functions, 3 were unused, 2 were only used here. As mentioned in the commit message, I don't think it's worth to print anything here, even with `verbose`, because it's not particularly interesting when it works. And when it *doesn't* work for some reason, just the rpath/runpath are not going to be enough to diagnose the issue.